tsa/create_pdf/main.py

```diff
@@ -0,0 +1,29 @@
+import io
+
+import pdfplumber
+from google.cloud import storage
+
+
+def get_pdf():
+    client = storage.Client()
+    bucket = client.get_bucket("tsa-throughput")
+    blob = bucket.get_blob("tsa-total-throughput-april-28-2024-to-may-4-2024.pdf")
```

Comment:
make pdf argument?

Reply:
yes, thanks for pointing this out and almost to this point. I need to build this out.